src/components/InputPassword.tsx

```diff
@@ -1,34 +1,59 @@
-import { Input } from "antd";
+import { TextField, IconButton, InputAdornment } from "@mui/material";
+import { useState } from "react";
+import { Visibility, VisibilityOff } from "@mui/icons-material";
 import styled from "styled-components";
 import colors from "../styles/colors";
 
-const InputPassword = styled(Input.Password)`
+const StyledTextField = styled(TextField)`
     background: ${(props) => (props.white ? colors.white : colors.lightNeutral)};
     box-shadow: 0px 2px 2px ${colors.shadow};
     border-radius: 4px;
-    border: none;
-    height: 40px;
+    
+    & .MuiOutlinedInput-root {
+        border-radius: 4px;
+        background: ${(props) => (props.white ? colors.white : colors.lightNeutral)};
+        
+        & fieldset {
+            border: none !important;
+        }
 
-    input {
-        background: ${colors.lightNeutral};
```

Comment:
Were the colors relevant in this exchange? If not, what was the reason for their removal?